setup/setup_validation_docker.sh

```diff
@@ -194,6 +208,9 @@ function install_all {
 
     # Install docker
     sudo yum -y install docker-ce python2-pip
+    sudo mkdir -p /etc/systemd/system/docker.service.d
```

Comment:
@david-richer-adsk @marlan16 Are we afraid of this? Should we enforce TLS too?

Reply:
I was also wondering and I asked the question during a scrum, but the answer was not at this time. If we add the support for tls, we will need to modify SEC4D to add the possibility to specify tls certificate when creating a new node.